examples/jsm/loaders/LDrawLoader.js

```diff
@@ -2080,8 +2102,24 @@ class LDrawLoader extends Loader {
 				material = loader.getMaterial( colorCode );
 				if ( material === null ) {
 
-					// otherwise throw an error if this is final opportunity to set the material
-					throw new Error( `LDrawLoader: Material properties for code ${ colorCode } not available.` );
+					// otherwise throw a warning if this is final opportunity to set the material
+					console.warn( `LDrawLoader: Material properties for code ${ colorCode } not available.` );
+
+					// And return the 'missing color' material
+					material = loader.missingColorMaterial;
+					material.name = "Missing material for color code '" + colorCode + "'";
+					material.userData.code = colorCode;
+
+					const edgeMat = loader.missingEdgeColorMaterial.clone();
+					edgeMat.name = "Missing material for color code '" + colorCode + "' - Edge";
+					edgeMat.userData.code = colorCode;
+
+					const condEdgeMat = loader.missingConditionalEdgeColorMaterial.clone();
+					condEdgeMat.name = "Missing material for color code '" + colorCode + "' - Edge";
+					condEdgeMat.userData.code = colorCode;
+
+					material.userData.edgeMaterial = edgeMat;
+					material.userData.edgeMaterial.userData.conditionalEdgeMaterial = condEdgeMat;
```

Comment:
`material.userData.edgeMaterial` can be changed just to `edgeMat`.

Reply:
Thanks, done.